ext/drmgr/drmgr.c

```diff
@@ -1026,8 +1097,18 @@ drmgr_bb_event(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
             drcontext, sizeof(void *) * local_info.quartet_count);
     }
 
-    res = drmgr_bb_event_do_instrum_phases(drcontext, tag, bb, for_trace, translating, pt,
-                                           &local_info, pair_data, quartet_data);
+    bool is_dups = false;
+    /*only true if drbbdup is in use */
```

Comment:
nit: missing space (and generally prefer full-sentence w/ punctuation)

Reply:
Done.